docs/source/upgrade-guide/index.md

```diff
@@ -186,6 +186,25 @@ becomes:
 import { slateBeforeEach } from '@plone/volto/cypress/support/helpers';
 ```
 
+### StoryBook 8
+
+StoryBook was upgraded from version 6 to 8 in core and in the project generator.
+This section is relevant if you have StoryBook stories in your project or add-on.
+The versions will be upgraded automatically using the `volto-update-deps` script.
+The configuration of your project must also be updated with the new one.
+Replace the `.storybook` folder in your project with this one:
+
+ `https://github.com/plone/volto/tree/main/packages/generator-volto/generators/app/templates/.storybook`
```

Comment:
I think we want to use the permalink for the latest commit, as the main branch may have changes in the future. Then again, maybe that would be a good thing so that the upgrade guide always links to the latest version of this folder? I'm not sure which is better.

In either case, a naked URL will be converted to a clickable link, and is preferable to use an inline literal. Choose one:

```suggestion
https://github.com/plone/volto/tree/main/packages/generator-volto/generators/app/templates/.storybook

=== OR ===

https://github.com/plone/volto/tree/5605131868689778bbdca0c3003a40cb9f153c1a/packages/generator-volto/generators/app/templates/.storybook
```

Reply:
Added permalink